test/integration/Makefile

```diff
@@ -52,7 +52,7 @@ delete-vcsim:
 	kubectl -n kube-system delete service vcsim
 
 deploy-ccm:
-	kubectl -n kube-system create configmap vsphere-cloud-config --from-file=vsphere.conf && \
+	kubectl -n kube-system create configmap cloud-config --from-file=vsphere.conf && \
```

Comment:
This was set to `vsphere-cloud-config` but the pod spec used for tests always referenced `cloud-config`:

https://github.com/kubernetes/cloud-provider-vsphere/blob/3ceddf092e38b5850185b5b9d93148bea3b9b115/manifests/controller-manager/vsphere-cloud-controller-manager-pod.yaml#L45-L48

Not sure how that was ever working.